types/carbon-components-react/lib/components/MultiSelect/MultiSelect.d.ts

```diff
@@ -27,10 +27,13 @@ export interface MultiSelectProps<T extends ListBoxBaseItemType = string> extend
     open?: boolean,
     selectionFeedback?: "fixed" | "top" | "top-after-reopen",
     useTitleInItem?: boolean,
+    placeholder?: string,
```

Comment:
The placeholder is marked as required. [FilterableMultiSelect.js#L78](https://github.com/carbon-design-system/carbon/blob/v10.7.4/packages/react/src/components/MultiSelect/FilterableMultiSelect.js#L78)

Reply:
Changed!